src/cursor/sys/unix.rs

```diff
@@ -1,42 +1,34 @@
+//! UNIX related logic to cursor manipulation.
+
 use std::io::{self, Write};
 
 use crate::input::{InputEvent, TerminalInput};
 use crate::utils::{
     sys::unix::{disable_raw_mode, enable_raw_mode, is_raw_mode_enabled},
     Result,
 };
-use crate::{csi, write_cout};
 
-pub(crate) fn get_cursor_position() -> Result<(u16, u16)> {
+/// Returns the cursor position (column, row).
+/// The counting starts from 0 were column 0 and row 0 is the top left.
```

Comment:
Empty doc comment line between these two lines. Also I'd use the same wording from the previous comment:

Top left cell is represented as `0, 0`.

Short and clear enough.